src/main/java/org/broadinstitute/hellbender/tools/copynumber/CondenseReadCounts.java

```diff
@@ -0,0 +1,203 @@
+package org.broadinstitute.hellbender.tools.copynumber;
+
+import htsjdk.samtools.util.BlockCompressedOutputStream;
+import htsjdk.samtools.util.IOUtil;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgram;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.CopyNumberProgramGroup;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.tools.copynumber.formats.collections.SimpleCountCollection;
+import org.broadinstitute.hellbender.tools.copynumber.formats.records.SimpleCount;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.gcs.BucketUtils;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+import org.broadinstitute.hellbender.utils.tsv.DataLine;
+import org.broadinstitute.hellbender.utils.tsv.TableColumnCollection;
+import org.broadinstitute.hellbender.utils.tsv.TableReader;
+import org.broadinstitute.hellbender.utils.tsv.TableWriter;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.OutputStreamWriter;
+import java.io.Writer;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+@CommandLineProgramProperties(summary = "condenses read-counts in .tsv format into a new file also in .tsv format",
+        oneLineSummary = "condenses read-counts in .tsv format into a new file also in .tsv formas" ,
+        programGroup = CopyNumberProgramGroup.class)
+public class CondenseReadCounts extends CommandLineProgram {
+
+    public static final String FACTOR_LONG_NAME = "factor";
+    public static final String FACTOR_SHORT_NAME = "f";
+    public static final String EXPECTED_BIN_LENGTH_LONG_NAME = "out-bin-length";
```

Comment:
Or maybe something including "filter" in the name.

Reply:
--emit-only-bin-of-length 1000?

Reply:
In any case this is a bit of a hack in my opinion .... perhaps it should be a range or min... considering that is totally possible for the input to have a mix of bins sizes... in the problem at hand  some bins have size ~950bp .... which sounds acceptable for output. I added this argument in case that we decide to exclude those we can do it quickly.

Reply:
I think verb names should be left to booleans.  Perhaps "emitted-bin-length"?

Reply:
Also, specifying a range might be good.  You could imagine getting gigantic bins, depending on your blacklist.  This could be a problem if ever the model wants to use bin length as a covariate.  Let's discuss for the final tool design.